docs/frame-kit.md

```diff
@@ -168,16 +168,21 @@ export default function Page() {
 **@Param**
 
 ```ts
-type Button = {
+type ButtonMetadata = {
   label: string;
   action?: 'post' | 'post_redirect';
 };
 
+type ImageMetadata = {
+  src: string;
```

Comment:
I am probably going to change this to `target` in the next PR, just to keep it clean with the other APIs.

Reply:
Or maybe not 🤔